src/shutdown/shutdown.c

```diff
@@ -373,10 +373,10 @@ int main(int argc, char *argv[]) {
 
         disable_coredumps();
 
-        log_info("Sending SIGTERM to remaining processes...");
+        log_notice("Sending SIGTERM to remaining processes...");
```

Comment:
why this?

these aren't particularly relevant msgs? i mean, the msgs that warn about left-over processes are relevant and should be beyond LOG_INFO, but not these ones?

Reply:
In shutdown.c, in the first "patch", I did move some log_info to log_notice to help to see the main step of the shutdown process. But maybe you are right about it, I drop the commit that change it.